scripts/build-packages-dist.sh

```diff
@@ -44,6 +44,10 @@ dirs=`echo "$targets" | sed -e 's/\/\/src\/\(.*\):npm_package/\1/'`
 # do this to ensure that the version placeholders are properly populated.
 for pkg in ${dirs}; do
   pkg_dir="${bazel_bin_path}/src/${pkg}/npm_package"
+  # Make all directories in the previous package output writable. Bazel by default
+  # makes tree artifacts and file outputs readonly. This causes permission errors
+  # when deleting the folder. To avoid these errors, we make all files writable.
+  chmod -R u+w ${pkg_dir}
```

Comment:
`rm -rf` can safely target a non-existent directory, but for `chmod` we need to make sure it exists.  So we should wrap it in a directory check.

```suggestion
  if [[ -d "$pkg_dir" ]]; then
    chmod -R u+w ${pkg_dir}
    rm -Rf ${pkg_dir}
  fi
```

Reply:
Yeah, that's why the CI is currently failing for this PR. Will fix!

Reply:
Fixed.